### ISSUE TITLE
TypeError: Object of type int64 is not JSON serializable

### ISSUE THREAD

**Issue by pya** [jupyterlab/jupyterlab-hdf5#47]
I can see the groups in the HDF5 file.
But I get `TypeError: Object of type int64 is not JSON serializable` when double-clicking on a table.
Python: 3.9.0
JupyterLab: Version 2.2.9
 jupyterlab_hdf: 0.4.1
The HDF5 file was created with PyTables with this datatype:

```
class Tab(tables.IsDescription):
    number = tables.Float64Col()
```

I can see all content with `h5ls`.

**Comment by telamonian** [comment]
This should be fixed by #49. I'll leave this issue open until there's an actual release that includes the fix

**Comment by telamonian** [comment]
This is now part of the [v0.5.0 release](https://github.com/jupyterlab/jupyterlab-hdf5/releases/tag/v0.5.0)
